packages/store/src/IStore.sol

```diff
@@ -15,11 +16,18 @@ import { ISliceErrors } from "./ISliceErrors.sol";
  * @notice IStore implements the error interfaces for each library that it uses.
  */
 interface IStore is
-  IStoreData,
+  IStoreRead,
+  IStoreWrite,
   IStoreRegistration,
   IStoreErrors,
   IFieldLayoutErrors,
   IPackedCounterErrors,
   ISchemaErrors,
   ISliceErrors
-{}
+{
+  /**
+   * @notice Returns the version of the Store contract.
+   * @return version The version of the Store contract.
+   */
+  function storeVersion() external view returns (bytes32 version);
```

Comment:
wonder if we should separate this into its own interface and have both `IStoreRead` and `IStoreWrite` inherit it as part of the protocol, or if it's fine just being here to signal "`IStore` is the protocol" and the rest are just "incomplete parts" of it

Reply:
basically wondering: is it valid to be able to read or write to a partial implementation of a store without also exposing a `storeVersion` (since the encoding may change between versions)?